justfile

```diff
@@ -29,40 +31,40 @@ install-operator:
   @echo "Installed the operator executable"
 
 # Run the Solidity tests using `forge test` command
-test-foundry:
-  forge test -vvv
+test-foundry testname=".\\*":
```

Comment:
Should we still have a run all tests command anyway?

Reply:
If no name is passed in it will use the default value which runs all tests